### DISCUSSION TITLE
Border between flat ButtonToggle?

### DISCUSSION THREAD

**Discussion by mtn090899** [quasarframework/quasar#15949] [General - Components / Directives / etc]
Hello, so I want to put borders between the button options within the Button Toggle component 

![image](https://github.com/quasarframework/quasar/assets/78068147/48ab3819-67a5-4fde-b94f-6f6a5c05d8e9)

I see that there is a gap in between the buttons, is it possible to thin this gap and change its color? Or any other manner to achieve a similar result?

Thank you

**Comment by pdanpdan** [comment]
Make a codepen

**Reply by mtn090899** [reply]
Not a codepen, but here is a quick makeup of the code effectively. The actual code I'm working on is a storybook. 
``` 
<div id="q-app">
 <q-btn-toggle 
  v-model="model"             
  options = [
    { label: 'Option 1', value: 'option1' },
    { label: 'Option 2', value: 'option2' },
  ],
  colo r= 'transparent',
  text-color = 'black',
  unelevated></q-btn-toggle>
</div> 
```

JS: 
``` 
import { ref } from 'vue'

const model = ref('option1')

**Comment by suleimanas** [comment]
you can always use css, and style `q-btn-toggle > q-btn`

**Reply by mtn090899** [reply]
Well I'm specifically trying to modify the space in between the buttons, and I can't seem to figure out the exact tags to modify for that little sliver of space, or if its able to be modified in the first place

**Comment by mtn090899** [comment]
Nevermind, got it!
